assets/javascripts/discourse/initializers/retort-init.js.es6

```diff
@@ -41,11 +57,93 @@ function initializePlugin(api) {
       title: 'retort.title',
       position: 'first'
     }
-  })
+  });
 
   api.attachWidgetAction('post-menu', 'clickRetort', function() {
     Retort.openPicker(this.findAncestorModel())
-  })
+  });
+
+  api.reopenWidget('post-menu', {
+    toggleWhoLiked() {
+      this.state.retortedUsers = [];
+      return this._super();
+    },
+
+    getWhoRetorted(emoji) {
+      const { attrs, state } = this;
+
+      return ajax(`/retorts/${attrs.id}/users`, {
```

Comment:
I'm a little uneasy with calling ajax from within the html method; really that should be done before the render is called, probably in `toggleWhoRetorted`. I think that may help mitigate some of the state manipulation we're doing here, since the state will be set from outside, before we render it.
```
toggleWhoRetorted(emoji) {
  if (emoji == this.state.retort) {
    this.state.retort = null
    this.state.retortedUsers = []
    this.scheduleRerender()
  } else {
    this.state.retort = emoji
    this.state.likedUsers = []

    ajax(..., data: { retort: emoji }).then((users) => {
      this.state.retortedUsers = users.map(avatarAtts)
      this.scheduleRerender()
    })
  }
}
```

Reply:
(I'd expect to only need `retort` (or `showRetortsFor`), and `retortedUsers` as additional things on the state to make this functionality work)